pkg/generate/app/cmd/newapp_test.go

```diff
@@ -615,6 +651,22 @@ func TestRunAll(t *testing.T) {
 				imageStreams = append(imageStreams, tp)
 			case *deploy.DeploymentConfig:
 				got["deploymentConfig"] = append(got["deploymentConfig"], tp.Name)
+				if podTemplate := tp.Template.ControllerTemplate.Template; podTemplate != nil {
+					for _, volume := range podTemplate.Spec.Volumes {
+						var srcType string
+						if volume.VolumeSource.EmptyDir != nil {
+							srcType = "EmptyDir"
+						} else {
+							srcType = "UNKNOWN"
+						}
+						got["volumes"] = append(got["volumes"], volume.Name+"♥"+srcType)
```

Comment:
That would be ideal, but it would also require us to change the type of "expected" and probably changing other tests and I don't think that's worth it.